AAI/AAIConnectProfile.md

```diff
@@ -259,18 +301,50 @@ the Broker.
 
         3.  Access tokens MAY contain non-GA4GH Claims directly in the access token.
 
+    3.  A Broker MAY issue [Self-Contained Passports](#term-self-contained-passport).
+
+        1.  Self-Contained Passports MAY be large and are only for use via POST within
+            GA4GH-compatible service endpoints.
+
+        2.  Self-Contained Passports MUST NOT be used with OAuth2 endpoints that require
+            an `Authorization` header.
```

Comment:
Not even if something else than the self-contained passport (for instance, a regular OAuth2 access token) is placed to the Authorization header?